src/log/test.cc

```diff
@@ -343,7 +343,8 @@ TEST(Log, Speed_nogather)
 
 TEST(Log, GarbleRecovery)
 {
-  static const char* test_file="/tmp/log_for_moment";
+  static char* test_name = "/tmp/log_for_moment.XXXX");
+  static char* test_file = mktemp(testname);
```

Comment:
i think we can just put
```c
char* test_name = "/tmp/log_for_moment.XXXX";
test_name = mktemp(test_name);
```

no need to make them `static`, and no need to use different variables: both of them are just names;